WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -226,4 +226,55 @@ d_melchisedechianism_holy_order = {
 	monthly_income = 2
 
 	mercenary_type = zealots_composition
-}	
\ No newline at end of file
+}
+
+c_priscillianism_holy_order = {
```

Comment:
Unrelated to all holy orders being duchy-level. For consistence and to avoid unforeseen issues they should be the same.